tendrl/gluster_integration/sds_sync/event_utils.py

```diff
@@ -5,7 +5,8 @@
 from tendrl.commons.utils.time_utils import now as tendrl_now
 
 
-def emit_event(resource, curr_value, msg, instance, severity):
+def emit_event(resource, curr_value, msg,
+               instance, severity, alert_notify=None):
```

Comment:
Default should be `False` for alert_notify

Reply:
ack